src/main/kotlin/org/wycliffeassociates/otter/jvm/persistence/database/AppDatabase.kt

```diff
@@ -30,7 +30,7 @@ class AppDatabase(
         dsl = DSL.using(sqLiteDataSource.connection, SQLDialect.SQLITE)
 
         // Check if the database file exists
-        val schemaFile = File(listOf("src", "main", "Resources", "CreateAppDb.sql").joinToString(File.separator))
+        val schemaFile = File(listOf("src", "main", "resources", "CreateAppDb.sql").joinToString(File.separator))
```

Comment:
I know this isn't relevant to your changes on this branch, but I realized yesterday that this should be changed to something more like: `File(ClassLoader.getSystemResource("CreateAppDb.sql"))`

Reply:
yeah, for sure